ckanext/dgu/schema.py

```diff
@@ -141,6 +141,8 @@
 
 update_frequency_options = ['never', 'discontinued', 'annual', 'quarterly', 'monthly']
 
+publication_delay_options = ['minimal', 'good', 'reasonable', 'extreme']
```

Comment:
Ah, I see now how you do it on the front end. I guess the concern is that these words defined twice. I think the canonical place should be schema.py and dataset_form.py gets them from there.